### ISSUE TITLE
bearer token

### ISSUE THREAD

**Issue by akilada** [jstakun/elastalert-ocp#3]
Hi,

The following bearer token PR hasn't been merged to elastalert master yet.
https://github.com/Yelp/elastalert/pull/3076

I'm bit curious how did you use the 'es_bearer' option.

Cos I'm facing the same issues with OCP ES 4.6.

Thanks

**Comment by StribPav** [comment]
Hi, Akilada. The OpenShift 4.6 is using the Open Distro for ElasticSearch.  The fork of ElasticSearch has an alert plugin and other interesting plugins.

**Comment by akilada** [comment]
Hey @StribPav 

Thanks for the update. I haven't used it before. Do you have any information about it?

Regards
Akila

**Comment by jstakun** [comment]
> Hi,
> 
> The following bearer token PR hasn't been merged to elastalert master yet.
> [Yelp/elastalert#3076](https://github.com/Yelp/elastalert/pull/3076)
> 
> I'm bit curious how did you use the 'es_bearer' option.
> 
> Cos I'm facing the same issues with OCP ES 4.6.
> 
> Thanks

Hello. I've created custom image which includes this PR to support bearer token.

**Comment by StribPav** [comment]
Hi @akilada. I didn`t study the question deep. It will be my next target.

**Comment by akilada** [comment]
> Hi @akilada. I didn`t study the question deep. It will be my next target.

Hi @StribPav

I looked into the openshift elasticsearch and cluster logging operator. It doesn't have the alert plugin. I'm going to follow up with Redhat.

**Comment by akilada** [comment]
> > Hi,
> > The following bearer token PR hasn't been merged to elastalert master yet.
> > [Yelp/elastalert#3076](https://github.com/Yelp/elastalert/pull/3076)
> > I'm bit curious how did you use the 'es_bearer' option.
> > Cos I'm facing the same issues with OCP ES 4.6.
> > Thanks
> 
> Hello. I've created custom image which includes this PR to support bearer token.

Thought so. I tested it on a custom docker image as well. It worked. I'm following up with Redhat on updating sgconfig/roles_mapping.

**Comment by StribPav** [comment]
@akilada Yes, the default docker image of the Openshift cluster operator doesn`t have an alert plugin.
